python/lsst/daf/persistence/baseYaml.py

```diff
@@ -0,0 +1,104 @@
+# This file is part of daf_persistence
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org/).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""This module defines YAML I/O for key lsst.daf.base classes."""
+
+import yaml
+
+import lsst.daf.base
+
+# YAML representers for key lsst.daf.base classes
+
+
+def dt_representer(dumper, data):
+    """Represent an lsst.daf.base.DateTime (as ISO8601-formatted string in UTC)
+    """
+    return dumper.represent_scalar(u'lsst.daf.base.DateTime',
+                                   data.toString(lsst.daf.base.DateTime.UTC))
```

Comment:
Are you sure?  I thought it handled it with T23:59:60 (and that there was a test for this).  I'll check later.  The problem with TAI is that it is not absolutely clear that it is in use (the only indicator is the lack of "Z"), and I was expecting the YAML to be human-readable.

Reply:
You were right that `lsst.daf.base.DateTime` doesn't handle leap seconds in UTC (because it defers to `gmtime`).  This is fixable with a few lines of code for past and near-future times (since we know the leap second table).  I'm not sure if it's worth doing.

Reply:
I think we should ditch UTC support from DateTime, but I suppose that's an RFC or RFD. In any case...please use astropy here if you want to save using UTC. But I'm keen on TAI; I think it's a much saner time standard and as long as we document what we're doing it'll be plenty human readable.